### ISSUE TITLE
Out of date tutorials

### ISSUE THREAD

**Issue by rapfamily4** [NVIDIAGameWorks/Falcor#407]
Tutorials are out of date.

For instance, in the second tutorial there's a `SharedPtr` which cannot be reached in any way. It also seems that `RenderPass::create()` is also not a thing anymore (I may be wrong).

The fourth tutorial also tells me to declare and initialize a series of variables, whose types are not recognized at all:
```
Scene::SharedPtr mpScene;
GraphicsProgram::SharedPtr mpProgram;
GraphicsState::SharedPtr mpGraphicsState;
RasterizerState::SharedPtr mpRasterState;
GraphicsVars::SharedPtr mpVars;
```
Since I'm a complete novice I don't know what to modify and where to look. I don't know if there are other outdated details in the fourth tutorial since I basically stopped at the declaration above.

Could you please update the tutorials? Thanks in advance.

**Comment by 625673575** [comment]
since 5.0, there is no turorial at all. it's more like a in-house research engine now

**Comment by bipul-mohanto** [comment]
This lack of beginner guides and out-of-date materials is challenging for newbies like me. Where can I get more updated tutorials? Is there any active developers' forum other than the git issue?

**Comment by yolk-pie-L** [comment]
Agreed. I'm a beginner, and I want to try out the Mogwai demo, but I can't find the script or scene files anywhere in the project. The tutorial isn’t very helpful. I don't know how to get started.

**Comment by ASDAlexander77** [comment]
if I were u I would look into RTXGI SDK as this is better and newer version of "Falcor" SDK
